gui/wxpython/psmap/g.gui.psmap.py

```diff
@@ -46,7 +46,10 @@ def main():
     from psmap.frame import PsMapFrame
 
     app = wx.App()
-    frame = PsMapFrame(parent=None)
+    frame = PsMapFrame(
+        parent=None,
+        title=_("Cartographic Composer - GRASS GIS"),
```

Comment:
Why adding GRASS GIS here? If it follows the same logic than the others, shouldn't it be `Cartographic Composer` only?

Reply:
These are the `g.gui.*` tools and the idea here is that they stand on its own, so they include GRASS GIS in their title unlike when the same component is started as part of the main GUI. This requires customizing the title (i.e., setting a separate title) in all `g.gui.*` (which is the line we are looking at), so it is more code than just keeping the same ones.

Unfortunately, this is not consistent with starting module GUI from command line, but that would be more work as there is no mechanism in place to set title from outside.